packages/opentelemetry-resources/src/ResourceImpl.ts

```diff
@@ -81,7 +83,7 @@ class ResourceImpl implements Resource {
       try {
         this._rawAttributes[i] = [k, isPromiseLike(v) ? await v : v];
       } catch (err) {
-        diag.debug("a resource's async attributes promise rejected: %s", err);
+        diag.debug('promise rejection for resource attribute: %s', err);
```

Comment:
nit: This whole try/catch here *could* be dropped. We don't ever expect there to be a `_rawAttributes` value now that can reject ... unless the user is poking into private attributes on a created `ResourceImpl`, which is naughty.  (As ever, I'm totally fine if you decide to skip any "nit"s from me.)

Reply:
i agree i'll remove it